### ISSUE TITLE
Agents and their names

### ISSUE THREAD

**Issue by MasoudJTehrani** [autonomousvision/carla_garage#69]
Good morning,
I hope you're doing well. 
In the pre-trained weights files, you have 

- lav/tfpp_02_05_withheld
- lav/aim_02_05_withheld
- leaderboard/tfpp_wp_all
- longest6/tfpp_all

I wanted to know the name of each agent. For example, which one is transfuser++, what is the agent of longest or lav/aim or lav/tfpp, or which 
one is interfuser.

**Comment by Kait0** [comment]
TF++ are lav/tfpp_02_05_withheld and longest6/tfpp_all
There are two models because on the lav benchmark it is not allowed to train with data from town 02 and 05.
The longest6 model trained with data from all towns.

leaderboard/tfpp_wp_all is a model called TF++ WP from the appendix of the paper and lav/aim_02_05_withheld is the reproduces AIM method from the appendix as well.

Interfuser is not part of our repository. We used the weights and code from the [interfuser_repository](https://github.com/opendilab/InterFuser) for our experiments.

**Comment by MasoudJTehrani** [comment]
Thank you very much. so I guess longest6 was the best-performing one yes?
Also what is the difference between the training seeds? like tfpp_all_0 or tfpp_all_2?
And plant_all_1 ?

**Comment by Kait0** [comment]
Yes, the longest6 model is the best-performing one.

Well, the difference is the training seed. 
Each model was trained with a different random seed. 
The last number is an index to distinguish them.
Otherwise, they are the same.

The Plant model is a reproduction of the privileged [Plant](https://arxiv.org/abs/2210.14222) model, though we have not used it in our paper.
See [here](https://github.com/autonomousvision/carla_garage/blob/leaderboard_1/docs/additional_features.md).
